pom.xml

```diff
@@ -209,4 +208,76 @@
             </build>
         </profile>
     </profiles>
+    
+    <build>
```

Comment:
I think, you should rather use a profile here, because the JavaDocs are cleaned up, even if i didn't change anything!
It's kind of a hassel, when during development you always see thousands of deleted files.
Check, if it's possible in a profile, to combine the two goals "aggregate" and "build"